src/int.rs

```diff
@@ -360,6 +542,22 @@ impl cmp::PartialOrd for I256 {
     }
 }
 
+impl ops::Neg for I256 {
+    type Output = I256;
+
+    fn neg(self) -> Self::Output {
+        handle_overflow(self.overflowing_neg())
+    }
+}
+
+impl ops::Not for I256 {
+    type Output = I256;
+
+    fn not(self) -> Self::Output {
+        I256(!self.0)
```

Comment:
What is the idea behind this logic? I found that this ! all the words, but what does this mean? Especially for the word that contains the signature.

Reply:
I'm not sure what you mean. This is a bit-wise not operation, so it just does a bit-wise not of all the words.

Reply:
I see. I just wasn't sure if we need to handle the signature bit in any special way. But I guess we don't.

Reply:
Since we use two's complement there is no special consideration for the sign bit. It behaves as the primitive signed integer types do.